fontir/src/ir.rs

```diff
@@ -2151,49 +2138,49 @@ mod tests {
     #[test]
     fn invalid_move_after_first_point() {
         // A point of type 'move' must be the first point in an (open) contour.
-        let mut builder = GlyphPathBuilder::new("test".into(), 0);
+        let mut builder = GlyphPathBuilder::new(0);
         builder.move_to((2.0, 2.0)).unwrap();
         builder.end_path().unwrap();
         // move_to after ending the current subpath is OK
         builder.move_to((3.0, 3.0)).unwrap();
         // but it's an error if we try to do move_to again
         let result = builder.move_to((4.0, 4.0));
 
-        assert!(result.is_err());
-        let Err(PathConversionError::MoveAfterFirstPoint { glyph_name, point }) = result else {
-            panic!("unexpected error: {:?}", result);
-        };
-        assert_eq!("test", glyph_name.as_str());
-        assert_eq!((4.0, 4.0), (point.x, point.y));
+        assert_eq!(
+            result,
+            Err(PathConversionError::MoveAfterFirstPoint {
+                point: (4.0, 4.0).into()
+            })
+        );
```

Comment:
Why did we drop the glyph name?

Reply:
because `PathConversionError` now only exists inside `BadGlyph`, which stores the glyph name.